aten/src/ATen/native/quantized/cpu/qlinear.cpp

```diff
@@ -855,7 +856,9 @@ at::Tensor PackedLinearWeightsOnednn::apply_impl(
       ideep::matmul_forward::prepare</*is_dynamic=*/false>(
           params, x, w, b, y,
           src_scales, weights_scales, dst_scales,
-          src_zero_point, dst_zero_point, 1.0f, 1.0f, op_attr);
+          src_zero_point, dst_zero_point, 1.0f, 1.0f, op_attr,
+          is_input_qint8 ? ideep::tensor::data_type::s8 : ideep::tensor::data_type::u8,
+          is_input_qint8 ? ideep::s8s8 : ideep::u8s8);
```

Comment:
nit: might be easier to name the variables for these two

Reply:
Done